### DISCUSSION TITLE
Let's earn the Galaxy Brain achievement

### DISCUSSION THREAD

**Discussion by xiaowoonly** [kingToolbox/WindTerm#1343] [General]
Galaxy Brain | Bronze 🥉 --> 8 accepted answers

Galaxy Brain | Silver 🥈 --> 16 accepted answers

Galaxy Brain | Gold 🥇 --> 32 accepted answers

**Comment by eternitywen** [comment]
Well... Here is my first answer. If you accept it then I am closer.

**Comment by eternitywen** [comment]
And this is my second answer.
